nuxeo-runtime/nuxeo-runtime/src/main/java/org/nuxeo/runtime/services/config/ConfigurationServiceImpl.java

```diff
@@ -172,12 +178,122 @@ public String getPropertiesAsJson(String namespace) throws IOException {
      * Returns true if a string starts with a namespace.
      *
      * @param string a string
-     * @param namespace
      * @since 10.3
      */
     protected static boolean startsWithNamespace(String string, String namespace) {
         int nl = namespace.length();
         return string.length() > nl && string.charAt(nl) == '.' && string.startsWith(namespace);
     }
 
+    /**
+     * @since 11.1
+     */
+    @Override
+    public Optional<String> getString(String key) {
+        return Optional.ofNullable(getDescriptors().get(key)).map(ConfigurationPropertyDescriptor::getValue);
```

Comment:
We should add blank behavior here directly. A blank value should be identical to an absent one.